### DISCUSSION TITLE
Hello from a Blockchain newbie

### DISCUSSION THREAD

**Discussion by Safwan1445** [Cyfrin/foundry-full-course-cu#2657] [General]
Hi Everyone, I'm thrilled to join this group! I'm completely new to blockchain development, with zero prior knowledge. I'm excited to learn and eager to dive into this fascinating world.
Looking forward to learning from each of you and contributing to the community as I grow. Thank you so much @PatrickAlphaC

**Comment by EngrPips** [comment]
Welcome aboard, and we wish you the best.
